configurations/.eslintrc.js

```diff
@@ -61,7 +61,7 @@ module.exports = {
         importOrder: [],
         importOrderSeparation: true,
         importOrderSortSpecifiers: true,
-        endOfLine: 'lf',
+        endOfLine: 'crlf',
```

Comment:
nooo lets keep lf, we keep changing it back and forth. Does the settings.json enforcing lf not help? (I just added this to main) lets keep lf as the standard